src/test/java/teammates/test/cases/datatransfer/TeamEvalResultTest.java

```diff
@@ -346,13 +346,6 @@ public void testAverageColumns() {
         assertEquals(Arrays.toString(expected2),
                 Arrays.toString(TeamEvalResult.averageColumns(input2)));
 
-        try {
```

Comment:
can you just explain to me why you removed this?

Reply:
@amarlearning Since I have removed `NSU` from isSanitized() check, no assertion error will be thrown, i.e it is accepted as valid input.